src/components/PopoverMenu/Popover.tsx

```diff
@@ -44,11 +44,11 @@ export default function PopoverDropdown(props: PopoverProps): JSX.Element {
       onClose={handleClose}
       anchorOrigin={{
         vertical: 'bottom',
-        horizontal: 'center',
+        horizontal: 'left',
       }}
       transformOrigin={{
         vertical: 'top',
-        horizontal: 'center',
+        horizontal: 'left',
```

Comment:
Is this going to always align the left edges?

Reply:
Oh sorry, I missed your comment on squeezing 👍

Reply:
Np :) yes, mui takes care of it. Added a story to capture this as well.